plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -106,6 +119,10 @@ var NodesTable = React.createClass({
         return UnitHealthUtil.getHealthSorting(node);
       }
 
+      if (prop === "region") {
+        return node.get("domain").fault_domain.region.name;
```

Comment:
🤔  I wonder if it would make sense to create a lens to get the region name /cc @nLight

Reply:
Nah... 
```javascript
const lens = Objektiv.attr("fault_domain").attr("region").attr("name");
lens.get(node.get("domain"));
```
I should invest some time to make its interface friendlier